src/main/java/ome/services/RawPixelsBean.java

```diff
@@ -111,7 +112,11 @@ public RawPixelsBean() {
      */
     public RawPixelsBean(boolean checking, String omeroDataDir) {
         this.diskSpaceChecking = checking;
-        this.omeroDataDir = omeroDataDir;
+        //omeroDataDir is no longer used by this class
```

Comment:
Is it also worth adding another constructor `public RawPixelsBean(boolean checking)`, maybe switch to this constructor in the Spring configuration and mark this old constructor as `@Deprecated`?